src/main/java/core/basesyntax/service/TransactionConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.TransactionConverter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TransactionConverterImpl implements TransactionConverter {
+    public static final String SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> records) {
+        return records.stream()
+                .skip(1)
```

Comment:
```suggestion
                .skip(OFFSET)
```
let's make it more visible